R/FilteredDataset-utils.R

```diff
@@ -43,7 +43,7 @@
 #' @examplesIf requireNamespace("MultiAssayExperiment")
 #' # MAEFilteredDataset example
 #' library(MultiAssayExperiment)
-#' data(miniACC)
+#' data(miniACC, package = "MultiAssayExperiment")
```

Comment:
Isn't `package` only necessary if the package isn't attached?

Reply:
Yes, this was a strange thing, I just noticed that there's a typo and reverted this change.

My environment didn't have `MultiAssayExperiment` so the typo was not detected before the merge.

Reply:
Corrected here: 511ede211560e99bfdac3eb0637f28cc1c74310b